tests/test_quadrupole.py

```diff
@@ -124,3 +124,35 @@ def test_tilted_quadrupole_multiple_batch_dimension():
     outgoing = segment(incoming)
 
     assert torch.allclose(outgoing.particles[0, 0], outgoing.particles[0, 1])
+
+
+def test_quadrupole_bmadx_tracking():
+    incoming_beam = torch.load("tests/resources/quadrupole_bmadx/incoming_beam.pt")
```

Comment:
@jp-ga I just ran into some vectorization issues when specifying multiple quadrupole strengths and then tracking (as we do in GPSR), can you add a test for this if its not elsewhere?

Reply:
@jp-ga has this been fixed?